_posts/2017-03-17-implementing-the-schema-registry.md

```diff
@@ -0,0 +1,40 @@
+---
+layout:     post
+title:      Implementing The Schema Registry
+date:       2017-02-08 12:00
+summary:    Keeping on top of an area of technology that is as rapidly moving as the big data ecosystem is hard.  Our data tribe share some of their resources for keeping up to date.
+category:   Data
+tags:       Data, Improvements
+author:     callum_leahy
+---
+
+![Enigma Machine](/images/enigma.jpg)
+
+## Introduction
+
+The platform currently used within the Gaming Promotions Squad largely relies on Apache Kafka. Kafka is a data streaming service which is used to send data from one section of the stack to another. Apache Kafka works with Apache Avro, a data serialization system used to encode Kafka. As a result of using Avro we need to use a Schema Registry Service, this service will store the various Avro schemas that we require in order to encode and decode messages.
+
```

Comment:
In the paragraph above, I'd change "system used to encode Kafka" to be "system used to encode event data placed on to Kafka